pwiz_tools/Skyline/TestUtil/AbstractUnitTestEx.cs

```diff
@@ -199,5 +203,25 @@ public static bool HasKoinaServer()
         {
             return !string.IsNullOrEmpty(KoinaConfig.GetKoinaConfig().Server);
         }
+
+        /// <summary>
+        /// Get a system resource string. Useful when checking that a test throws an expected error message but the text comes from the OS or .NET.
+        /// Use an ID from https://github.com/ng256/Tools/blob/main/MscorlibMessage.md
+        /// </summary>
+        /// <example>GetSystemResourceString("IO.FileNotFound_FileName", "SomeFilepath")</example>
+        public string GetSystemResourceString(string resourceId, params object[] args)
+        {

```

Comment:
Did I miss an existing function in Skyline to do this?

Reply:
Probably not. This is a great addition, but I doubt we have been testing this specifically for messages thrown by .NET